ci-operator/templates/openshift/installer/cluster-launch-installer-ovirt-e2e.yaml

```diff
@@ -294,6 +294,17 @@ objects:
             --junit-dir /tmp/artifacts/junit -f -
         }
 
+        function run-csi-tests() {
+          for url in $(curl -s https://api.github.com/repos/ovirt/csi-driver-operator/contents/manifests | jq '.[].download_url'); do
```

Comment:
Please change ovirt/csi-driver-opeartor to openshift/ovirt-csi-driver-operator
Next we need the link of the image under test.

Reply:
> Please change ovirt/csi-driver-opeartor to openshift/ovirt-csi-driver-operator
Done

> Next we need the link of the image under test.

Can you explain how this will fix the error ?